packages/sanity/src/core/form/inputs/files/__workshop__/UploadPlaceholderStory.tsx

```diff
@@ -0,0 +1,25 @@
+import {Button, Card, Container, Flex} from '@sanity/ui'
+import React from 'react'
```

Comment:
I'm not sure if this is the new way of doing but do we want to add things to the workshop when we're trying to push to the next branch? I thought that it was just a development tool.

But maybe it makes sense to make sure that people can use this part of the workshop the next time they have to do something related to file / image?

Reply:
It's not a new way of doing it, workshop stories have been pushed to `next` since the workshop was introduced! It is a development tool, but it's good to push workshop stories so everyone on the team can work on and test the components in the workshop :)

Reply:
Might be really interesting to have a meeting showing off how you set it up and how you usually do dev with it since I don't think I ever used that tool :) Would you mind doing it?

Reply:
Yes, of course! 👍

Reply:
I'll schedule a meeting then! Probably more towards the end of this week 🙏